### DISCUSSION TITLE
Drag and drop Datagrid columns

### DISCUSSION THREAD

**Discussion by tshailesh-vmw** [vmware/clarity#5606] [Ideas]
Currently we cannot drag and drop data grid columns. This will be useful for report generation using the data present in the datagrid. User can already hide/show columns and with drag and drop column the saved data/ generated report will be as per user's choice.

Just asking if there is plan for this feature to be implemented or can this be implemented?

**Comment by Jinnie** [comment]
Your request has been considered and there was some work done in this direction. But, as you can verify in this issue https://github.com/vmware/clarity/issues/1771 we decided to put the Angular implementation on hold, in favor of the new Clarity Core datagrid component, that's under development.
